packages/component-header/src/components/UniversalNavbar/Search/index.js

```diff
@@ -48,8 +26,32 @@ const Search = () => {
     if (open) inputRef.current.focus();
   }, [open]);
 
+  const handleSearch = (e) => {
+    const form = e.target;
+    e.preventDefault();
+    /**
+     * Issue: Callback not currently available
+     * We need to ensure dataLayer events are being logged correctly
+     * Soluton might be in GA4 settings with a form event targeting the element ID
+     *
+     * This solution does not guarantee the event is logged before the page
+     * redirects.
+     * Preventing form submission with arbitrary timeout is always bad, but this
+     * may be small enough to not degrade the experience
+     *
+     * TODO: UDS-1612
+     */
+    trackGAEvent({
+      ...SEARCH_GA_EVENT,
+      text: e.target.elements.q.value,
+    });
+    setTimeout(() => {
+      form.submit();
+    }, 100);
+  };
+
   const handleChangeVisibility = () => {
-    setOpen(prevState => {
+    setOpen((prevState) => {
```

Comment:
Approved with or without this comment. Are our prettier setting somehow different? Single props do not get wrapped in () or should we update prettier settings?

Reply:
Good find. Let's not consider this blocking this issue since it itself is a blocker. Can be resolved elsewhere.